gateway/src/apicast/policy/maintenance_mode/policy.lua

```diff
@@ -0,0 +1,37 @@
+-- This is a simple policy. It allows you reject incoming requests with a
+-- specified status code and message.
+-- It's useful for maintance periods or to temporarily block an API
+
+local _M = require('apicast.policy').new('Maintenance mode')
+
+local tonumber = tonumber
+local new = _M.new
+
+
+function _M.new(configuration)
+  local policy = new(configuration)
+
+  -- Set some default values
+  policy.status_code = 503
+  policy.message = "503 Service Unavailable - Maintenance"
+
+  if configuration then
+    policy.status_code = tonumber(configuration.status) or policy.status_code
+    policy.message = configuration.message or policy.message
+  end
+
+  return policy
+end
+
+function _M:access()
+
+local status = self.status_code
+local msg = self.message
+
+  ngx.status = status
+  ngx.say(msg)
+
+  return ngx.exit(ngx.status)
+end
+
+return _M
```

Comment:
Github is complaining because there isn't a newline here.
Same with the rest of files included in this PR.

Reply:
Mmm is this still complaining? I'm fairly sure my git removes the last newline of a file. (I need to check this)

Reply:
@alanmoran I think there *needs* to be a new line.

Reply:
@kevprice83 I added it and its there locally but I think git or github removing it.